### PR TITLE
Update eggdrop docker image

### PR DESCRIPTION
**Pull request by vanosg**

### CODE REVIEW

**Comment by yosifkit** [comment]
LGTM, Build test of #2053; 152946a (`eggdrop`):

```console

$ bashbrew build eggdrop:1.8
Building bashbrew/cache:26464c439cf5ab7d62f30b900e03b8bd1a4aea577ab072d372653de83cf63a39 (eggdrop:1.8)
Tagging eggdrop:1.8

$ test/run.sh eggdrop:1.8
testing eggdrop:1.8
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build eggdrop:1.6
Building bashbrew/cache:7e503242d2d3eb6954825069dd904034de2a7d57daccd4467746d7cdf3249761 (eggdrop:1.6)
Tagging eggdrop:1.6

$ test/run.sh eggdrop:1.6
testing eggdrop:1.6
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed

```

```diff
diff --git a/eggdrop_latest/Dockerfile b/eggdrop_1.6/Dockerfile
similarity index 100%
rename from eggdrop_latest/Dockerfile
rename to eggdrop_1.6/Dockerfile
diff --git a/eggdrop_latest/docker.tcl b/eggdrop_1.6/docker.tcl
similarity index 100%
rename from eggdrop_latest/docker.tcl
rename to eggdrop_1.6/docker.tcl
diff --git a/eggdrop_latest/entrypoint.sh b/eggdrop_1.6/entrypoint.sh
similarity index 100%
rename from eggdrop_latest/entrypoint.sh
rename to eggdrop_1.6/entrypoint.sh
diff --git a/eggdrop_develop/Dockerfile b/eggdrop_1.8/Dockerfile
similarity index 79%
rename from eggdrop_develop/Dockerfile
rename to eggdrop_1.8/Dockerfile
index 099988e..fcd4b67 100644
--- a/eggdrop_develop/Dockerfile
+++ b/eggdrop_1.8/Dockerfile
@@ -6,12 +6,11 @@ RUN adduser -S eggdrop
 # grab su-exec for easy step-down from root
 RUN apk add --no-cache 'su-exec>=0.2'
 
-ENV EGGDROP_SHA256 40c3e41cece00b3e3741bd1cd92d8e8bf3fd0adee2cad54bcb04cfc26f38f4a0
-ENV EGGDROP_COMMIT 1753372f439849ccc4e6b692f90991282b6daf28
+ENV EGGDROP_SHA256 e5cfae339d176377353d2f28e2a6ae27c4d3ea8b099653407a4f6cfd5b329972
+ENV EGGDROP_COMMIT cfd5abc83d19fb3ea14c51a484e8761cbd368968
 
 RUN apk add --no-cache tcl tcl-dev wget ca-certificates make tar gpgme bash build-base \
-  && wget https://github.com/eggheads/eggdrop/archive/1753372f439849ccc4e6b692f90991282b6daf28.tar.gz -O develop.tar.gz\
-  && sha256sum develop.tar.gz \
+  && wget https://github.com/eggheads/eggdrop/archive/$EGGDROP_COMMIT.tar.gz -O develop.tar.gz\
   && echo "$EGGDROP_SHA256  develop.tar.gz" | sha256sum -c - \
   && tar -zxvf develop.tar.gz \
   && rm develop.tar.gz \
diff --git a/eggdrop_develop/docker.tcl b/eggdrop_1.8/docker.tcl
similarity index 100%
rename from eggdrop_develop/docker.tcl
rename to eggdrop_1.8/docker.tcl
diff --git a/eggdrop_develop/entrypoint.sh b/eggdrop_1.8/entrypoint.sh
similarity index 100%
rename from eggdrop_develop/entrypoint.sh
rename to eggdrop_1.8/entrypoint.sh
```

File: library/eggdrop [hunk 1]
```diff
@@ -1,10 +1,10 @@
 Maintainers: Geo Van O <geo@eggheads.org> (@vanosg)
 GitRepo: https://github.com/eggheads/eggdrop-docker.git
 
-Tags: 1.6.21, 1.6, stable, latest
-GitCommit: def15a99429271388c9f2958dcc1a825f729e3d3
-Directory: 1.6
-
-Tags: 1.8.0, 1.8, develop
```
Comment (yosifkit) [inline library/eggdrop]: Did you mean to drop all the other tags for `1.8` and `1.6`?

**Comment by vanosg** [comment]
No! Guess I tried adapting to that generation script a little too quickly, thank you for that catch.

**Comment by tianon** [comment]
LGTM :+1:

Build test of #2053; 6549185 (`eggdrop`):

```console

$ bashbrew build eggdrop:1.8
Building bashbrew/cache:26464c439cf5ab7d62f30b900e03b8bd1a4aea577ab072d372653de83cf63a39 (eggdrop:1.8)
Tagging eggdrop:1.8
Tagging eggdrop:1.8.0
Tagging eggdrop:develop

$ test/run.sh eggdrop:1.8
testing eggdrop:1.8
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build eggdrop:1.6
Building bashbrew/cache:7e503242d2d3eb6954825069dd904034de2a7d57daccd4467746d7cdf3249761 (eggdrop:1.6)
Tagging eggdrop:1.6
Tagging eggdrop:1.6.21
Tagging eggdrop:stable
Tagging eggdrop:latest

$ test/run.sh eggdrop:1.6
testing eggdrop:1.6
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed

```

**Pull request merged**